### PR TITLE
Find minknow output files that were renamed in the latest version

### PR DESCRIPTION
**Pull request by ssjunnebo**
The output files from minknow have been renamed. TACA should now be able to find them.

### CODE REVIEW

File: taca/analysis/analysis_nanopore.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def process_run(run_dir, nanoseq_sample_sheet, anglerfish_sample_sheet):
         qc_run = False
 
     logger.info('Processing run: {} as a {}'.format(run_dir, 'QC run' if qc_run else 'non-QC run'))
-    summary_file = os.path.join(run_dir, 'final_summary.txt')
+    summary_file = glob.glob(run_dir + '/final_summary*.txt')[0]
```
Comment (aanil) [inline taca/analysis/analysis_nanopore.py:46]: Are there multiple summary and report files being generated? Just wondering why it requires a wild card.

Comment (ssjunnebo) [reply taca/analysis/analysis_nanopore.py:46]: Nah, but they changed the name to final_summary_<some ID>.txt and report_<some ID>.txt. Before it was just final_summary.txt and report.txt

Comment (aanil) [reply taca/analysis/analysis_nanopore.py:46]: Hmm, strange way to rename the files.

**Review by aanil — APPROVED** [review]
👍  Looks good!

**Pull request merged**